foundation-models/openai/src/test/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClientTest.java

```diff
@@ -84,7 +85,7 @@ void apiVersion() {
     verify(exactly(2), postRequestedFor(anyUrl()).withoutQueryParam("api-version"));
   }
 
-  private static Runnable[] chatCompletionCalls() {
+  private static Runnable[] errorHandlingCalls() {
```

Comment:
why the renaming?

Reply:
`chatCompletionCalls` is defined below already, I canot have the same name twice